sub-packages/bionemo-llm/src/bionemo/llm/lightning.py

```diff
@@ -258,6 +259,8 @@ def __init__(
         self._data_step = data_step
         self._forward_step = forward_step
         self.model_transform = model_transform
+        self.train_ppl = torchmetrics.text.Perplexity(ignore_index=-100)
+        self.valid_ppl = torchmetrics.text.Perplexity(ignore_index=-100)
```

Comment:
Would be great to be configurable once the PR has matured.

Reply:
For sure, I think we sould make an `ESM2LightningModule` with a lot of these args pre-defined where we can then attach metrics in an obvious way